packages/transaction-messages/src/__typetests__/durable-nonce-typetests.ts

```diff
@@ -72,4 +85,43 @@ type V0TransactionMessage = Extract<TransactionMessage, { version: 0 }>;
         // @ts-expect-error Should not be a v0 message.
         newMessage satisfies TransactionMessageWithDurableNonceLifetime & V0TransactionMessage & { some: 1 };
     }
+
+    // It prepends the nonce instruction to the transaction message.
+    {
+        const feePayer = null as unknown as Address;
+        const message = pipe(
+            createTransactionMessage({ version: 0 }),
+            m => setTransactionMessageFeePayer(feePayer, m),
+            m => appendTransactionMessageInstruction(null as unknown as InstructionA, m),
+            m => setTransactionMessageLifetimeUsingDurableNonce(mockNonceConfig, m),
+        );
+
+        message satisfies CompilableTransactionMessage;
```

Comment:
I was going to say maybe wherever you have `Address` make it `Address<'abc'>` to make sure that the specific address flows through, but looks like `CompilableTransactionMessage` doesn't have a type parameter to pass on to `TransactionMessageWithFeePayer`. Grumble.